fjord/feedback/jinja2/feedback/android_thanks.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'android_feedback' %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'android-feedback' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Firefox for Android Feedback') }}{% endblock %}
```

Comment:
Should the title of this match the title of the feedback form?

Also, we should include the same L10N message here.